pkg/kafka/auth/options.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import "flag"
+
+const (
+	suffixAuthentication = ".authentication"
+
+	// Kerberos configuration options
+	kerberosPrefix            = ".kerberos"
+	suffixKerberosServiceName = ".service-name"
+	suffixKerberosRealm       = ".realm"
+	suffixKerberosUseKeyTab   = ".use-keytab"
+	suffixKerberosUserName    = ".username"
+	suffixKerberosPassword    = ".password"
+	suffixKerberosConfig      = ".config-file"
+	suffixKerberosKeyTab      = ".keytab-file"
+
+	defaultAuthentication      = "none"
```

Comment:
nit: related to my previous comment, remove this and just pass `authTypes[0]` to flag definition